Typescript local import

I made a quick test project to try out the new Typescript support structure looks like this.

```
.
├── functions
│   ├── jokeTypescript.js
│   └── jokeVanilla.js
├── netlify.toml
├── package.json
├── src_functions
│   ├── jokeTypescript.ts
│   ├── jokeVanilla.js
│   └── utils
│       └── direction.ts
└── yarn.lock
```

when I try to import something local like this

```
import { Direction } from './utils/direction';
```

in jokeTypescript.ts, I get an error in the build like this

```
ERROR in ./jokeTypescript.ts
Module not found: Error: Can't resolve './utils/direction' in '/Users/matt/sandbox/nl-test/src_functions'
 @ ./jokeTypescript.ts 6:0-46 19:21-30
```

external imports like `import * as request from 'request-promise-native';` seem to work though

any thoughts?

--- a/lib/build.js
+++ b/lib/build.js
@@ -47,6 +47,9 @@ function webpackConfig(dir, additionalConfig) {
 
   var webpackConfig = {
     mode: "production",
+    resolve: {
+      extensions: ['.js', '.mjs', '.ts']
+    },
     module: {
       rules: [
         {

--- a/lib/build.js
+++ b/lib/build.js
@@ -48,7 +48,7 @@ function webpackConfig(dir, additionalConfig) {
   var webpackConfig = {
     mode: "production",
     resolve: {
-      extensions: ['.js', '.mjs', '.ts']
+      extensions: ['.wasm', '.mjs', '.js', '.json', '.ts']
     },
     module: {
       rules: [

